src/proc/namer.rs

```diff
@@ -91,6 +88,52 @@ impl Namer {
             }
         }
 
+        for (handle, var) in module.global_variables.iter() {
+            let name = self.call_or(&var.name, "global");
+            output.insert(NameKey::GlobalVariable(handle), name);
+        }
+
+        for (handle, constant) in module.constants.iter() {
+            let label = match constant.name {
+                Some(ref name) => name,
+                None => {
+                    use std::fmt::Write;
+                    // Try to be more descriptive about the constant values
+                    temp.clear();
+                    match constant.inner {
+                        crate::ConstantInner::Scalar {
+                            width: _,
+                            value: crate::ScalarValue::Sint(v),
+                        } => write!(temp, "const_{}i", v),
+                        crate::ConstantInner::Scalar {
+                            width: _,
+                            value: crate::ScalarValue::Uint(v),
+                        } => write!(temp, "const_{}u", v),
+                        crate::ConstantInner::Scalar {
+                            width: _,
+                            value: crate::ScalarValue::Float(v),
+                        } => write!(
+                            temp,
+                            "const_{}_{}f",
+                            v.abs().trunc(),
+                            (v.fract() * 100.0) as i8
```

Comment:
The naming output seems a bit off when using this, e.g. https://github.com/gfx-rs/naga/pull/461/files#diff-332e4d2098a2ab03271f1d2d73b2ac4b8b8612fa8510c95573e7fea94a46ba65R48

Could we do something like use `f64`'s `Display` impl and substitute `.` with `_`, `-` with `n`?

Reply:
is `const_0_5f` wrong? I thought that's the best we can do

Reply:
I expected something like `const_0_05f = 0.05` there (an extra zero), especially because we also have `const_0_50f = 0.5` above

Reply:
oh snap, you are right, that's totally wrong!